aws-iot-securityprofile/aws-iot-securityprofile.json

```diff
@@ -129,6 +131,28 @@
             "minimum": 0,
             "maximum": 65535
           }
+        },
+        "Number": {
+          "description": "The numeral value of a metric.",
+          "type": "number"
+        },
```

Comment:
This comment is actually related to "Count" field. Didn't notice that earlier by why is its data type "string"?
https://docs.aws.amazon.com/iot/latest/apireference/API_MetricValue.html#iot-Type-MetricValue-count

Reply:
The datatype for count is string because the json-schema does not support Long datatypes. The work around is to use string and translate the value. Anton has a comment in the code for it, pasting it here: 

"// The MetricValue is a String in the CFN model because the json-schema doesn't support Long numbers.
        // This is a known issue and some existing services are already using Strings as the workaround.
        // For example, AWS::ElasticLoadBalancing::LoadBalancer HealthCheck
        // https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-elb-health-check.html"